src/connection_pool.rs

```diff
@@ -118,7 +118,7 @@ where
             thread::spawn(move || rebalancer_loop(max_connections, protected_data_clone2, rebalancer_clone2, rebalancer_log_clone));
 
         let pool = ConnectionPool {
-            protected_data: protected_data,
+            protected_data,
```

Comment:
hm, interesting shortcut.

Reply:
This was a new one to me as well. I'm ambivalent about it, but I figured I'd try it out and see how I like for a while.